aip/general/0009.md

```diff
@@ -120,14 +120,18 @@ A Network API exposed by a Google service. Most of these are hosted on the
 `googleapis.com` domain. It does not include other types of APIs, such as client
 libraries and SDKs.
 
-### IaC
+### Declarative Clients
 
-Short for Infrastructure-as-code, IaC describes a category of clients that
+Declarative Clients describes a category of clients that
```

Comment:
Maybe we could say they are also known as Infrastructure as Code (IaC)?

Reply:
There are also other types of clients with this behavior, such as controllers, data-driven testing tools, and state-based policy constraint checkers.

Reply:
Done